content/general/limits.textile

```diff
@@ -124,6 +167,26 @@ Bandwidth is the amount of data transferred through "messages":/glossary#message
 
 If the bandwidth rate is exceeded, a "global instantaneous rate limit":#global-rates will be applied to message delivery and publishing. The hourly and total bandwidth limits are "time-based":#time. If the hard limit is hit for those, message delivery and publishing will be blocked until the following hour or month.
 
+h4(#bandwidth-notifications). Bandwidth notifications
+
+Bandwidth limit notifications are split into hourly and monthly notifications. 
+
+h5(#bandwidth-hourly-notifications). Hourly notifications
+
+There are three types of hourly bandwidth limit notifications you may receive:
+
+* @messages.hourly.warning.data@ - when you reach 80% of your pre-paid quota.
+* @messages.hourly.soft.data@ - when your pre-paid quota has been exceeded.
```

Comment:
"when your pre-paid quota has been exceeded" -> "when the soft limit on the account, derived from the pre-paid quota, has been exceeded"